File: android/ffmpeg-kit-android-lib/src/main/java/com/lgcns/ffmpegkit/MediaInformationSessionCompleteCallback.java
package com.lgcns.ffmpegkit;

/**
 * <p>Callback function that is invoked when an asynchronous <code>MediaInformation</code> session
 * has ended.
 */
@FunctionalInterface
public interface MediaInformationSessionCompleteCallback {

    /**
     * <p>Called when a media information session has ended.
     *
     * @param session media information session
     */
    void apply(final MediaInformationSession session);

}

File: android/ffmpeg-kit-android-lib/src/main/java/com/lgcns/ffmpegkit/MediaInformationSessionCompleteCallback.kt
package com.lgcns.ffmpegkit

/**
 *
 * Callback function that is invoked when an asynchronous `MediaInformation` session
 * has ended.
 */
fun interface MediaInformationSessionCompleteCallback {
    /**
     *
     * Called when a media information session has ended.
     *
     * @param session media information session
     */
    fun apply(session: MediaInformationSession?)
}